python/guide/module/gluon/parameters.md

```diff
@@ -184,7 +198,7 @@ net[0].weight.data()[0]
 
 ## Tied Parameters
 
-In some cases, we want to share model parameters across multiple layers. For instance when we want to find good word embeddings we may decide to use the same parameters both for encoding and decoding of words. We discussed one such case when we introduced [Blocks](model-construction.md). Let's see how to do this a bit more elegantly. In the following we allocate a dense layer and then use its parameters specifically to set those of another layer.
+In some cases, we want to share model parameters across multiple layers. For instance, when we want to find good word embeddings we may decide to use the same parameters both for encoding and decoding of words. In the code below, we allocate a dense layer and then use its parameters specifically to set those of another layer.
 
 ```{.python .input  n=14}
 net = nn.Sequential()
```

Comment:
exampe -> example

Reply:
Capital for start of comments, and remove . at end.

Reply:
Fixed